src/XMLSecurityDSig.php

```diff
@@ -152,7 +152,7 @@ private function getXPathObj()
      */
     public static function generateGUID($prefix='pfx')
     {
-        $uuid = md5(uniqid(mt_rand(), true));
+        $uuid = sha1(random_bytes(32));
```

Comment:
That makes more sense yes, will updated (already done in https://github.com/robrichards/xmlseclibs/pull/101)